css/style.css

```diff
@@ -5,7 +5,6 @@ body {
   background-position: center;
   background-repeat: no-repeat;
   background-size: cover;
-
```

Comment:
Please make sure to delete undesirable spaces so your code would be cleaner

Reply:
- [ ] [ OPTIONAL] Your comment is polite and you give a reason here. Good job, But I think that you give some explanation to make the author understand why is important to have clean code :smile: